packages/active-campaign-client/src/helpers/manageListSubscription.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { acClient } from '../clients/activeCampaignClient';
+
+export async function manageListSubscription(
+  cognitoUserId: string,
+  listName: string,
+  action: 'subscribe' | 'unsubscribe'
+): Promise<APIGatewayProxyResult> {
+  try {
+    const contactId = await acClient.getContactByCognitoId(cognitoUserId);
+    if (!contactId) {
+      return {
+        statusCode: 404,
+        body: JSON.stringify({ message: 'Contact not found' }),
+      };
+    }
```

Comment:
To make this function testable should be indipendent to cognito otherwise test cannot work properly